docs/make.jl

```diff
@@ -20,7 +23,9 @@ makedocs(
                 "Grassmann" => "manifolds/grassmann.md",
                 "Hyperbolic space" => "manifolds/hyperbolic.md",
                 "Lorentzian manifold" => "manifolds/lorentz.md",
+                "Multinoial matrices" => "manifolds/multinomial.md",
```

Comment:
```suggestion
                "Multinomial matrices" => "manifolds/multinomial.md",
```

Reply:
More interestingly, the docs page is still empty...I was too fast I fear, still checking.